nevergrad/optimization/base.py

```diff
@@ -414,7 +414,7 @@ def _update_archive_and_bests(self, candidate: p.Parameter, loss: tp.FloatLoss)
         if self.pruning is not None:
             self.archive = self.pruning(self.archive)
 
-    def ask(self) -> p.Parameter:
+    def ask(self, ignore_constraints: bool = False) -> p.Parameter:
```

Comment:
please don't change the ask and tell API

Reply:
Fixed.